### PR TITLE
UI tweaks

### PR DESCRIPTION
**Pull request by g-ortuno**
Small UI tweaks. Please review @jyasskin @scheib 

device-2015-03-18-130733

### CODE REVIEW

**Comment by jyasskin** [comment]
Here too, say more about the changes in your commit messages. You can rebase to fix this.

File: app/src/main/res/values/color.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="primary">#0F9D58</color>
```
Comment (jyasskin) [inline app/src/main/res/values/color.xml]: Comment where these colors came from. They should probably be a palette from http://www.google.com/design/spec/style/color.html#color-color-palette, but I don't see them there.

**Comment by jyasskin** [comment]
LGTM, with 1 comment.

File: app/src/main/res/values/color.xml (hunk 1 above)
Comment (g-ortuno) [inline app/src/main/res/values/color.xml]: Done.

**Pull request merged**